### ISSUE TITLE
[Bug]: Cannot make problem sets public

### ISSUE THREAD

**Issue by dqnykamp** [Doenet/DoenetApps#2994]
### Before submitting

- [x] I searched [existing issues](https://github.com/Doenet/DoenetApps/issues?q=is%3Aissue) and did not find a duplicate.

### Affected area

Editor

### Summary

There doesn't appear to be a way to satisfy the Syntax checks or Accessibility checks for problem sets, so they cannot be made public. 

### Steps to reproduce

1. Create a problem set
2. Try to make it public
3. You can't pass the checks

### Expected behavior

Those checks should recurse to the documents and pass when each document passes.

### Actual behavior

You can't get it to pass the tests

### Relevant logs

```shell

```

**Comment by cqnykamp** [comment]
Fixed by #2998
